### ISSUE TITLE
[How to use] Improve Documentation

### ISSUE THREAD

**Issue by KeremKurban** [BlueBrain/NeuroTS#102]
### Describe the problem

Some parameters are partially explained in the paramters and distributions page. For example growth_method has several arguments but there is no link to redirect to extensive documentation or to its algorithm. With so many parameters, its hard to debug when we see something wrong and dont know exactly what is going on under the hood.

### Show what you tried to do.

```Python
Trying to synthesize pyramidal neurons having parameters like

CA1sp_PC,axon.orientation.values.direction.mean,3.1415

means we have angle of pi radians but does it mean this one goes 180 degrees opposite of orientation.nrrd value there? or is this orientation of soma to apical ? Need illustrations , maybe examples on cortical column
```

### Describe what you expected.

i assume this is atlas-agnostic and should be opposite of apical direction for typical PC. but if it has a multipolar dendrite, then i have no idea what will happen. or in a case of interneuron, there is no apical section, then what would happen ? 

### neurots Version in Use

3.5.0

**Comment by arnaudon** [comment]
yes, if you read the line above, it says `pia_constraint` where pia means `orientation.dot([0, 1, 0])`, so it will be an inverted apical tree indeed!

**Comment by arnaudon** [comment]
I should maybe add a bit more doc on that part, its fairly new, so I assign this to myself
